gherkin-languages.json

```diff
@@ -0,0 +1,3514 @@
+{
+  "af": {
+    "and": [
+      "* ",
```

Comment:
I also noticed this in gherkin documentation: https://cucumber.io/docs/gherkin/reference/#gherkin-dialects
what is meant by `*`. Is it like a wild card, catch-all?
in that case `Given`, `When`, `Then`, `And`, `But` all are equal. And contextually there is no difference (this note probably is more related to `cabbage` library on how to handle step definitions and how important it is to use correct step type. It seems that there is 0 importance to it, only easier to read for the eye because is more similar to plain English)

Reply:
I'm really not a gherkin expert, but yes it seems like it can be used to avoid redundancy in feature files (cf [https://stackoverflow.com/a/27363033](https://stackoverflow.com/a/27363033)). I removed it from the keywords list in Keywords. cleanup_list/1 but maybe it's better to leave it there, what do you think ?

Reply:
I think that the primary concern for this library should be compatibility with original spec. So if gherkin/cucumber supports it, we should as well. So `*` should be supported.